assets/src/web-stories-block/css/variables.css

```diff
@@ -7,7 +7,7 @@
   --ws-list-aspect-ratio: 75%;
   --ws-overlay-text-color: #fff;
   --ws-overlay-text-lh: 1.3;
-  --ws-story-min-width: 185px;
+  --ws-story-min-width: 235px;
```

Comment:
Any details on why this was changed?

Reply:
After I'd removed the `min-width` property that was there on the story poster element, the stories were looking too small for the viewports, hence increased the min-width to something more reasonable for all the view ports.